cheat.php

```diff
@@ -148,7 +150,17 @@
 		'{normal} - Difficulty: {yellow}' . GetNameForDifficulty( $Zone )
 	);
 
-	$SkippedLagTime = floor( curl_getinfo( $c, CURLINFO_TOTAL_TIME ) - curl_getinfo( $c, CURLINFO_STARTTRANSFER_TIME ) );
+	if ( !isset( $RepositoryScriptHash ) || $LocalScriptHash === $RepositoryScriptHash )
```

Comment:
`isset` is not necessary as you assign it at the top.

Reply:
isset is to check if the variable has yet to be set, so we can skip later with ===.

Reply:
You assign the variable when the script starts, it can't be undefined.

Reply:
`	if ( $LocalScriptHash === $RepositoryScriptHash )
	{
		$RepositoryScriptHash = GetRepositoryScriptHash( );
	}
	else
	{
		Msg('-- {lightred}Repository script has been modified. Make sure to check it for updates.');
	}`

This will prevent it from generating the repository hash, ever... No? Since $RepositoryScriptHash will be undefined.

Reply:
Add `$RepositoryScriptHash = '';` at top.

Sorry I missed it last time.

Reply:
For this to work the way you want, I need to add $RepositoryScriptHash = $LocalScriptHash; instead of ''.

Reply:
More precisely:

`$LocalScriptHash = GetLocalScriptHash( );
$RepositoryScriptHash = $LocalScriptHash;`

Reply:
You are right.

`$RepositoryScriptHash = $LocalScriptHash = GetLocalScriptHash()`